bats/back-compat.bats

```diff
@@ -14,6 +14,7 @@ teardown() {
     for testdir in */; do
         cd "$testdir"
         dolt status
+        skip "Backwards compatibility broken with unique tags (v0.16.0)"
```

Comment:
these working when we migrated automatically ([CI](https://jenkins.awsdev.ld-corp.com/job/liquidata-inc/job/dolt/job/PR-491/16/))